### ISSUE TITLE
Linting fails with 0.25 (delegate.rb:87:in `method_missing')

### ISSUE THREAD

**Issue by agross** [brigade/haml-lint#232]
```
undefined method `children' for false:HamlLint::ParsedRuby
C:/Ruby/lib/ruby/2.3.0/delegate.rb:87:in `method_missing'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/tree/tag_node.rb:237:in `existing_attributes'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/tree/tag_node.rb:38:in `has_hash_attribute?'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/linter/inline_styles.rb:9:in `visit_tag'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:40:in `safe_send'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:19:in `visit'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `block in visit_children'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `each'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `visit_children'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:24:in `visit'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `block in visit_children'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `each'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `visit_children'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:24:in `visit'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `block in visit_children'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `each'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `visit_children'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:24:in `visit'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `block in visit_children'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `each'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `visit_children'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:24:in `visit'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `block in visit_children'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `each'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:30:in `visit_children'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/haml_visitor.rb:24:in `visit'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/linter.rb:28:in `run'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/runner.rb:84:in `block in collect_lints'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/runner.rb:83:in `map'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/runner.rb:83:in `collect_lints'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/runner.rb:119:in `process_file'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/runner.rb:108:in `block in process_files'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/runner.rb:107:in `each'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/runner.rb:107:in `process_files'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/runner.rb:132:in `report'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/runner.rb:23:in `run'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/cli.rb:106:in `scan_for_lints'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/cli.rb:51:in `act_on_options'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/cli.rb:23:in `run'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/rake_task.rb:97:in `run_cli'
C:/Ruby/lib/ruby/gems/2.3.0/gems/haml_lint-0.25.0/lib/haml_lint/rake_task.rb:87:in `block in define'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/task.rb:250:in `block in execute'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/task.rb:250:in `each'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/task.rb:250:in `execute'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-funnel-0.19.0/lib/rake/funnel/integration/progress_report.rb:48:in `call'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-funnel-0.19.0/lib/rake/funnel/integration/progress_report.rb:48:in `block (4 levels) in create_patch'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-funnel-0.19.0/lib/rake/funnel/integration/progress_report.rb:48:in `call'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-funnel-0.19.0/lib/rake/funnel/integration/progress_report.rb:48:in `block (4 levels) in create_patch'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-funnel-0.19.0/lib/rake/funnel/tasks/timing.rb:89:in `call'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-funnel-0.19.0/lib/rake/funnel/tasks/timing.rb:89:in `block (5 levels) in benchmark'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-funnel-0.19.0/lib/rake/funnel/support/timing/statistics.rb:22:in `benchmark'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-funnel-0.19.0/lib/rake/funnel/tasks/timing.rb:81:in `block (2 levels) in benchmark'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-funnel-0.19.0/lib/rake/funnel/tasks/timing.rb:88:in `block (4 levels) in benchmark'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/task.rb:194:in `block in invoke_with_call_chain'
C:/Ruby/lib/ruby/2.3.0/monitor.rb:214:in `mon_synchronize'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/task.rb:187:in `invoke_with_call_chain'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/task.rb:216:in `block in invoke_prerequisites'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/task.rb:214:in `each'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/task.rb:214:in `invoke_prerequisites'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/task.rb:193:in `block in invoke_with_call_chain'
C:/Ruby/lib/ruby/2.3.0/monitor.rb:214:in `mon_synchronize'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/task.rb:187:in `invoke_with_call_chain'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/task.rb:180:in `invoke'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:152:in `invoke_task'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:108:in `block (2 levels) in top_level'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:108:in `each'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:108:in `block in top_level'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:117:in `run_with_threads'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:102:in `top_level'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:80:in `block in run'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:178:in `standard_exception_handling'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:77:in `run'
C:/Ruby/lib/ruby/gems/2.3.0/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
C:/Ruby/bin/rake:22:in `load'
C:/Ruby/bin/rake:22:in `<main>'
Report this bug at https://github.com/brigade/haml-lint/issues

To help fix this issue, please include:
- The above stack trace
- Haml-Lint version: 0.25.0
- Haml version: 4.0.7
- RuboCop version: 0.48.1
- Ruby version: 2.3.3
rake aborted!
haml-lint failed with exit code 70
```

My config was unchanged after updating haml-lint. Still using haml 4, FYI.

**Comment by trotzig** [comment]
I'll have a look at this.

**Comment by trotzig** [comment]
If possible, could you do some bisection to figure out what file + line in that file the error is coming from? I have a fix in mind, but I want to make sure it's the right one. It would be good to have a repro case. (ping @mensfeld as well, since you saw the same issue).

A workaround in the meantime is to disable the `InlineStyles` linter.

**Comment by trotzig** [comment]
Thanks @agross for the report! This is fixed in 0.25.1.

**Comment by agross** [comment]
Thanks for the quick fix!
